opentracing-web-servlet-filter/src/main/java/io/opentracing/contrib/web/servlet/filter/TracingFilter.java

```diff
@@ -148,10 +139,10 @@ public void doFilter(ServletRequest servletRequest, ServletResponse servletRespo
         if (servletRequest.getAttribute(SERVER_SPAN_CONTEXT) != null) {
             chain.doFilter(servletRequest, servletResponse);
         } else if (isTraced(httpRequest, httpResponse)){
-            SpanContext extractedContext = tracer.extract(Format.Builtin.HTTP_HEADERS,
+            SpanContext extractedContext = GlobalTracer.get().extract(Format.Builtin.HTTP_HEADERS,
                     new HttpServletRequestExtractAdapter(httpRequest));
 
-            final Span span = tracer.buildSpan(httpRequest.getMethod())
+            final Span span = GlobalTracer.get().buildSpan(httpRequest.getMethod())
```

Comment:
This is a very good question. I believe that users should be aware that `GlobalTracer` is used underneath. 

Storing the first result of `GlobalTracer#get()` vs using current registered tracer. Bot of these has cons and pros. I believe it is better to be consistent here and go with the simplest approach at the moment.

cc @bhs @objectiser